src/Compilers/CSharp/Portable/Symbols/Source/CustomModifierUtils.cs

```diff
@@ -154,5 +154,15 @@ internal static bool HasInAttributeModifier(this ImmutableArray<CustomModifier>
         {
             return modifiers.Any(modifier => !modifier.IsOptional && ((CSharpCustomModifier)modifier).ModifierSymbol.IsWellKnownTypeInAttribute());
         }
+
+        internal static bool HasIsExternalInitModifier(this ImmutableArray<CustomModifier> modifiers)
+        {
+            if (modifiers.IsDefault)
```

Comment:
>if (modifiers.IsDefault) [](start = 12, length = 24)

Why do we need this check here, but not in the helper above. What is different?

Reply:
This feels wrong, It feels unexpected for a symbol to expose modifiers as ```ImmutableArray<CustomModifier>.Default```. I think we should understand the root cause.  

---
In reply to: [417705806](https://github.com/dotnet/roslyn/pull/43275#discussion_r417705806) [](ancestors = 417705806)